lib/features/currency/payin.dart

```diff
@@ -93,10 +109,20 @@ class Payin extends HookWidget {
                 textBaseline: TextBaseline.alphabetic,
                 children: [
                   Flexible(
-                    child: AutoSizeText(
-                      formattedAmount,
-                      style: Theme.of(context).textTheme.displayMedium,
+                    child: AutoSizeText.rich(
+                      TextSpan(
+                        children: [
+                          TextSpan(text: formattedAmount),
+                          TextSpan(
+                            text: decimalPaddingHint.value,
+                            style: TextStyle(
+                              color: Colors.grey.shade600,
```

Comment:
should a custom grey for hint text be added to `ColorScheme` instead?

Reply:
I think so. That will help with light/dark mode as well.